webpack.config.js

```diff
@@ -105,7 +105,10 @@ const GutenbergBlocksConfig = {
 				use: {
 					loader: 'babel-loader?cacheDirectory',
 					options: {
-						presets: [ '@wordpress/default' ],
+						presets: [ '@wordpress/babel-preset-default' ],
```

Comment:
I'm a bit confused with this. In [`@wordpress/babel-preset-default` docs](https://www.npmjs.com/package/@wordpress/babel-preset-default) it recommends using `"presets": [ "@wordpress/default" ]`.

Reply:
Hmm ya, essentially I just noticed that the full form was being used everywhere in Gutenberg (but not that the docs were just recommending the shorthand).  Incidentally, I asked about this in [#core-editor slack](https://wordpress.slack.com/archives/C02QB2JS7/p1565703369143500) and I learned about [shorthands for presets)(https://babeljs.io/docs/en/presets#preset-shorthand) - which I didn't know about.  

So the original was sufficient.  However, I'd prefer the longform because imo it's a bit clearer what package is providing the preset.  Happy to change back though?

Reply:
Oh, I didn't know about this either. Thanks for posting the explanation!

> So the original was sufficient. However, I'd prefer the longform because imo it's a bit clearer what package is providing the preset. Happy to change back though?

Agree, let's keep the long name.